Assets/Plugins/CountlySDK/Input/InputObserverResolver.cs

```diff
@@ -8,7 +8,7 @@ public static class InputObserverResolver
         public static IInputObserver Resolve()
         {
             IInputObserver observer = null;
-#if UNITY_ANDROID || UNITY_IOS
+#if !UNITY_EDITOR && (UNITY_ANDROID || UNITY_IOS)

```

Comment:
This change should be reverted. It should use the same observer even in the editor